Sofa/Component/Constraint/Projective/src/sofa/component/constraint/projective/OscillatorConstraint.inl

```diff
@@ -49,13 +49,15 @@ OscillatorConstraint<TDataTypes>*  OscillatorConstraint<TDataTypes>::addConstrai
 
 
 template <class TDataTypes> template <class DataDeriv>
-void OscillatorConstraint<TDataTypes>::projectResponseT(const core::MechanicalParams* /*mparams*/, DataDeriv& res)
+void OscillatorConstraint<TDataTypes>::projectResponseT(const core::MechanicalParams* /*mparams*/ /* PARAMS FIRST */, DataDeriv& res,
+    std::function<void(DataDeriv&, const unsigned int)> clear)
```

Comment:
Why do you add a function parameter here? The function is called only once

Reply:
it is called twice, once in projectResponse and in projectJacobianMatrix
But it was backported at it is, I will clean this responseT stuff